src/inverseRobot.js

```diff
@@ -23,8 +23,24 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  let swappedObject = [];
+  const mySet = new Set();
+  const arrRobot = Object.values(robot);
+
+  for (const key of arrRobot) {
+    mySet.add(key);
+  }
+
+  if (arrRobot.length === mySet.size) {
+    swappedObject = Object.fromEntries(
+      Object.entries(robot).map(v => v.reverse()));
+  } else {
+    return null;
+  }
```

Comment:
have this check first, and then change key and value exchange step
and just need one condition, 
```suggestion
  if (uniqueValues !== arrRobot.length) {
    return null;
  }
```